assets/js/state/sagas/user.js

```diff
@@ -24,7 +24,9 @@ export function* performLogin({ payload: { username, password } }) {
     yield call(storeRefreshToken, refreshToken);
     yield put(setUserAsLogged());
   } catch (error) {
-    yield put(setAuthError({ error: error.message }));
+    yield put(
+      setAuthError({ message: error.message, code: error.response.status })
```

Comment:
Could this somehow fails on some other error?
Like network error where we don't have a status code.
Would it go to undefined?

Reply:
nice catch!